src/dataset.js

```diff
@@ -201,18 +201,16 @@ export class Dataset {
     }
 
     /**
-     * Returns items in the dataset based on the provided parameters. The returned object
-     * has the following structure:
-     *
-     * ```javascript
-     * {
-     *     items, // Array|String|Buffer based on chosen format parameter.
-     *     total, // Number
-     *     offset, // Number
-     *     count, // Number
-     *     limit, // Number
-     * }
-     * ```
+     * @typedef DatasetContent
```

Comment:
does JSDoc `typedef` have support for generics? such as `DatasetContent<T>`.

Reply:
JSDoc does not support generics. But TS certainly does add them with the `@template` tag. Can you, please, write down all your ideas. You'll get a chance to get back to it later.